cell_types_utils.R

```diff
@@ -1,4 +1,14 @@
 #!/usr/bin/env Rscript
+
+# Define global variables 
+# list of unlabelled cell types
+# NB: keep these relevant to the tools' output
+unlabelled = c("unassigned", "Unassigned", "unknown",
```

Comment:
Apologies for this question, I'm having a bit of trouble teasing apart the logic in this big PR. 

Is the set of unlabelled values configurable by the scripts and functions that use this variable? Obviously the user running the scripts needs to be able to override these.

Reply:
@pinin4fjords yes, I meant to add config file parameters for those but forgot that - will add them (tomorrow). Otherwise these are just parameters that I made global variables as they are used in pretty much every script